actors/miner/src/lib.rs

```diff
@@ -216,6 +225,69 @@ impl Actor {
         })
     }
 
+    /// Returns the owner address
+    fn get_owner(rt: &mut impl Runtime) -> Result<GetOwnerReturn, ActorError> {
+        rt.validate_immediate_caller_accept_any()?;
+        let state: State = rt.state()?;
+        let owner = get_miner_info(rt.store(), &state)?.owner;
+        Ok(GetOwnerReturn { owner })
+    }
+
+    /// Returns the worker address
+    fn get_worker(rt: &mut impl Runtime) -> Result<GetWorkerReturn, ActorError> {
+        rt.validate_immediate_caller_accept_any()?;
+        let state: State = rt.state()?;
+        let worker = get_miner_info(rt.store(), &state)?.worker;
+        Ok(GetWorkerReturn { worker })
+    }
+
+    /// Returns the control addresses
+    fn get_controls(rt: &mut impl Runtime) -> Result<GetControlsReturn, ActorError> {
+        rt.validate_immediate_caller_accept_any()?;
+        let state: State = rt.state()?;
+        let controls = get_miner_info(rt.store(), &state)?.control_addresses;
+        Ok(GetControlsReturn { controls })
+    }
+
+    /// Returns the miner's sector size
+    fn get_sector_size(rt: &mut impl Runtime) -> Result<GetSectorSizeReturn, ActorError> {
+        rt.validate_immediate_caller_accept_any()?;
+        let state: State = rt.state()?;
+        let sector_size = get_miner_info(rt.store(), &state)?.sector_size;
+        Ok(GetSectorSizeReturn { sector_size })
+    }
+
+    /// Returns the miner's total funds locked as pre_commit_deposit
+    fn get_pre_commit_deposit(
```

Comment:
Should we add `_total` to this and the other methods below? There could later be per-sector versions.

Reply:
Yeah, seems reasonable.